tensorflow/compiler/xla/pjrt/tpu_client.cc

```diff
@@ -246,6 +247,11 @@ GetTpuDevices(
 
 StatusOr<std::shared_ptr<PjRtClient>> GetTpuClient(
     int max_inflight_computations, absl::Duration init_retry_timeout) {
+  Status tpu_library_finder = tensorflow::tpu::FindAndLoadTpuLibrary();
+  if (tpu_library_finder != Status::OK()) {
+    LOG(INFO) << tpu_library_finder;
```

Comment:
Do we need to LOG(INFO) this? I think you can just use `TF_RETURN_IF_ERROR` if not

Reply:
If we don't LOG(INFO) this, then when it does not work, it will not log the error message when we TF_CPP_MIN_LOG_LEVEL=0 run it. I thought it is useful for now to keep it?

Reply:
It should end up being logged here: https://github.com/google/jax/blob/main/jax/_src/lib/xla_bridge.py#L263
Is this not the case?

Reply:
Let's leave this for now so we can get this in actually, and then we can remove it later if it's actually logging twice.

Reply:
AAah, you are right! it is actually logging twice. I changed it! Thanks!